File: 1. First year/laboratory_exercises/04_exercise/01_task.py
def square_area(a):
    area = a * a
    return f"Лицето на квадрата е: {area}"


def rectangle_area(a, b):
    area = a * b
    return f"Лицето на правоъгълника е: {area}"


def triangle_are(a, b):
    area = (a * b) / 2
    return f"Лицето на правоъгълния триъгълник е: {area}"


figure = input()
if figure == "квадрат":
    a = float(input("Въведете страна: "))

    print(square_area(a))
elif figure == "правоъгълник":
    a = float(input("Въведете a: "))
    b = float(input("Въведете b: "))

    print(rectangle_area(a, b))
elif figure == "правоъгълен триъгълник":
    a = float(input("Въведете a: "))
    b = float(input("Въведете b: "))

    print(triangle_are(a, b))

File: 1. First year/02_semester/laboratory_exercises/03_exercise/01_task.c
#include <stdio.h>
#include <math.h>

void squareArea(double a) {
    double result = a * a;

    printf("Square Area: %.2lf", result);
}

void rectangleArea(double a, double b) {
    double result = a * b;

    printf("Rectangle Area: %.2lf", result);
}

void triangleArea(double a, double b) {
    double result = (a * b) / 2;

    printf("Triangle Area: %.2lf", result);
}

void circleArea(double r) {
    double result = 3.14 * pow(r, 2);

    printf("Circle Area: %.2lf", result);
}

int main() {
    int num;

    scanf("%d", &num);

    if (num == 1) {
        double a, b;

        scanf("Enter a: %lf", &a);
        scanf("Enter b: %lf", &b);

        squareArea(a, b);
    } else if (num == 2) {
        double a, b;

        scanf("Enter a: %lf", &a);
        scanf("Enter b: %lf", &b);

        rectangleArea(a, b);
    } else if (num == 3) {
        double a, b;

        scanf("Enter a: %lf", &a);
        scanf("Enter b: %lf", &b);

        triangleArea(a, b);
    } else if (num == 4) {
        double r;

        scanf("Enter r: %lf", &r);

        circleArea(r);
    }

    return 0;
}
